src/Text/Pandoc/Readers/HTML.hs

```diff
@@ -673,6 +687,9 @@ inline = pTagText <|> do
         "var" -> pCodeWithClass "var" "variable"
         "span" -> pSpan
         "math" -> pMath False
+        "input" 
+          | Just x <- lookup "type" attr,
+            x == "checkbox" -> asks inListItem >>= guard >> pCheckbox
```

Comment:
Will this work?
```hs
| Just "checkbox" <- lookup "type" attr
```
if so it is cleaner

Reply:
Hmmm that'd be neat, I'll check. I'm new to guards in case statements (didn't even know it could be done!)

Reply:
or maybe just `| lookup "type" attr == Just "checkbox"`?